packages/custom-templated-path-webpack-plugin/package.json

```
@@ -6,8 +6,9 @@
 	"license": "GPL-2.0-or-later",
 	"keywords": [
 		"wordpress",
+		"gutenberg",
 		"webpack",
-		"webpack-plugin"
+		"plugin"
```

webpack-plugin seems a [common label](https://www.npmjs.com/search?q=keywords:webpack-plugin) as well

```suggestion
		"plugin",
		"webpack-plugin",
```